waspc/src/Wasp/Generator/WriteFileDrafts.hs

```diff
@@ -31,6 +31,9 @@ import Wasp.Util (Checksum)
 -- It is smart when writing, so it doesn't write file drafts that are already written on the disk from before.
 synchronizeFileDraftsWithDisk :: Path' Abs (Dir ProjectRootDir) -> [FileDraft] -> IO ()
 synchronizeFileDraftsWithDisk dstDir fileDrafts = do
+
+  return $! assertDestPathsAreUnique fileDrafts
```

Comment:
After reading through the code, I see we are mixing "dest" and "dst". Given the existing format is "dst", how about we keep that same style in this file too?

```suggestion
  return $! assertDstPathsAreUnique fileDrafts
```